templates-assets/templates/NotificationAAR_RADDalt/i18n/sl.json

```diff
@@ -32,5 +33,6 @@
   "delegate_list_2":"svoj osebni dokument;",
   "delegate_list_3":"svojo davčno številko;",
   "delegate_list_4":"kopijo ali izvirnik davčne številke in identifikacijski dokument osebe, ki je podala pooblastilo.",
-  "delegate_list_5":"un documento che attesti il tuo ruolo di legale rappresentante."
+  "delegate_list_5":"un documento che attesti il tuo ruolo di legale rappresentante.",
```

Comment:
here the sentence is "un documento che attesti il ruolo di legale rappresentante del delegante."